graphite/graphite.go

```diff
@@ -100,6 +101,15 @@ func readTraceback(resp *http.Response) (*[]string, error) {
 // DefaultClient is the default HTTP client for requests.
 var DefaultClient = &http.Client{
 	Timeout: time.Minute,
+	Transport: &http.Transport{
+		MaxIdleConnsPerHost: 0,
```

Comment:
It appears you can also set it to < 0 with the same effect, without changing the global setting. This may need some thought.